shell/common/input_events_unittests.cc

```diff
@@ -124,7 +124,9 @@ static void TestSimulatedInputEvents(
     for (int i = 0, j = 0; i < num_events; j += 1) {
       double t = j * frame_time;
       while (i < num_events && delivery_time(i) <= t) {
-        ShellTest::DispatchFakePointerData(shell.get());
+        // Use a different x every time for the pointer data converter to
+        // generate non-empty events.
```

Comment:
Should we move the "use a different x" trick to `DispatchFakePointerData`'s implementation? I don't feel strongly about this, feel free to keep as is.